docs/building/MlNetMklDeps/MlNetMklDeps.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/01/nuspec.xsd">
   <metadata>
     <id>MlNetMklDeps</id>
-    <version>0.0.0.9</version>
+    <version>0.0.0.11</version>
```

Comment:
Can you explain the versioning story here?

Here we have `0.0.0.11`. In the version.md we have `0.0.0.10`. and in the Versions.props we have `0.0.0.12`.

Why are these all different?

Reply:
They should all be the same and the Version.props is correct. This has now been fixed.